content/configuration/source.md

```diff
@@ -170,22 +170,30 @@ sources:
           backendServiceValid: true
           # If true, notifies about Ingress resources with invalid TLS secret reference.
           tlsSecretValid: true
+      
+      # Describes namespaces configuration for every Kubernetes resources you want to watch or exclude.
+      # These namespaces are applied to every resource specified in the resources list.
+      # However, every specified resource can override this by using its own namespaces object.
+      namespaces:
+        # Include contains a list of allowed Namespaces.
+        # It can also contain a regex expressions:
+        #  `- ".*"` - to specify all Namespaces.
+        include:
+          - ".*"
+        # Exclude contains a list of Namespaces to be ignored even if allowed by Include.
+        # It can also contain a regex expressions:
+        #  - "test-.*" - to specify all Namespaces with `test-` prefix.
+        # exclude: []
 
       # Describes the Kubernetes resources you want to watch.
       resources:
```

Comment:
could you also remove the other:
```yaml
          namespaces:
            include:
              - ".*"
```
IMO it is worth because of 2 reason:
- shorter example
- it the same as the top one, so it can be confusing why we repeat that in our official example.

Reply:
Yup, they should've been removed. I missed them.